routers/router.go

```diff
@@ -238,6 +238,11 @@ func initAPI() {
 	beego.Router("/api/webauthn/signin/begin", &controllers.ApiController{}, "Get:WebAuthnSigninBegin")
 	beego.Router("/api/webauthn/signin/finish", &controllers.ApiController{}, "Post:WebAuthnSigninFinish")
 
+	beego.Router("/api/mfa/setup/initiate", &controllers.ApiController{}, "POST:TwoFactorSetupInitiate")
+	beego.Router("/api/mfa/setup/verify", &controllers.ApiController{}, "Post:TwoFactorSetupVerify")
+	beego.Router("/api/mfa/setup/enable", &controllers.ApiController{}, "POST:TwoFactorSetupEnable")
+	beego.Router("/api/mfa", &controllers.ApiController{}, "DELETE:TwoFactorDelete")
```

Comment:
why not use POST + `/api/delete-mfa`?

Reply:
Will `Post` work?

![image](https://user-images.githubusercontent.com/3787410/236358388-bfeca4bb-d8eb-47b6-98a8-03415cdebaac.png)